src/components/Button/index.ts

```diff
@@ -0,0 +1 @@
+export { Button } from './Button'
```

Comment:
don't we have a global index that exports all of the components?

How does a user of the lib use this component?

I'd expect to write

```ts
import {Button} from 'lib'
```

and not


```ts
import {Button} from 'lib/components/Button'
```